mbtiles/src/mbtiles.rs

```diff
@@ -272,9 +397,44 @@ impl Mbtiles {
         }))
     }
 
-    /// Get a tile from the database
+    /// Retrieves a single tile from the database by its coordinates.
+    ///
+    /// Returns the raw tile data as a byte vector if the tile exists at the given
+    /// zoom level and x/y coordinates. Returns `None` if no tile exists at those
+    /// coordinates.
+    ///
+    /// # Coordinate System
+    ///
+    /// Coordinates use the XYZ tile scheme where:
+    /// - `z` is the zoom level (0-30)
+    /// - `x` is the column (0 to 2^z - 1)
+    /// - `y` is the row in XYZ format (0 at top, increases southward)
     ///
-    /// See [`Mbtiles::get_tile_and_hash`] if you do need the hash
+    /// Note: `MBTiles` files internally use TMS coordinates (0 at bottom), but this
+    /// method handles the conversion automatically.
```

Comment:
I am so glad that crates.io recently added this feature 😉 
So much nicer

```suggestion
    /// > [!NOTE]
    /// > MBTiles files internally use [osgeos' Tile Map Service](https://wiki.openstreetmap.org/wiki/TMS) coordinates (0 at bottom).
    /// > This method handles the conversion automatically as maplibre/mapbox expect this.
```